src/app/public/src/modules/code-block/code-block.component.html

```diff
@@ -1,6 +1,17 @@
 <div class="stache-code-block">
-  <div #codeFromContent class="stache-code-input"><ng-content></ng-content></div>
-  <pre class="stache-code-output"><code
-    [ngClass]="getClassName()"
-    [innerHTML]="output"></code></pre>
-</div>
\ No newline at end of file
+  <div #codeFromContent class="stache-code-input">
+    <ng-content></ng-content>
+  </div>
+  <div stacheClipboard [attr.clipboardTriggerButton]="copyCodeButton">
+    <pre class="stache-code-output"><code [ngClass]="getClassName()" [innerHTML]="output"></code></pre>
+    <button
+      aria-label="copy to clipboard"
```

Comment:
We are going to want to add this label through the SkyAppResources pipe, and store the value in the `resources_en_US.json` file.  For accessibility we may end up having to support several translations for these fields.